src/index.js

```diff
@@ -0,0 +1,216 @@
+import fs from 'fs'
+import zlib from 'zlib'
+import path from 'path'
+import request from 'request'
+import changeCase from 'change-case'
+
+import { createHMAC, getSauceEndpoint, toString, getParameters } from './utils'
+import {
+    PROTOCOL_MAP, DEFAULT_OPTIONS, SYMBOL_INSPECT,
+    SYMBOL_TOSTRING, SYMBOL_ITERATOR, TO_STRING_TAG
+} from './constants'
+
+export default class SauceLabs {
+    constructor (options) {
+        this._options = Object.assign({}, DEFAULT_OPTIONS, options)
```

Comment:
I'm missing the possibility to use it if you're behind a `proxy`. This was in the previous version and isn't here anymore (or I need new 👓 )
This should be there before we release this version because a some SL customers also use a proxy

Reply:
The package is using the request package which allows handling proxy settings via env variables. I will add a note to the readme